File: grade.py
#Python Program to Take in the Marks of 5 Subjects and Display the Grade.

a=[]

for i in range(0,5):
	b=int(input("enter the marks"))
	a.append(b)
print(a)
percentage=sum(a)/5

if(percentage >= 65):
	print("secures grade A")

elif(65 > percentage >= 50):
	print("secures grade B")

elif(50 > percentage >= 35):
	print("secures grade C")

else:
	print("fail")

File: grade.c
#include<stdio.h>

void main()
{
	int a[5],i,sum=0;
	float avg;
	for(i=0;i<5;i++)
	{
		printf("enter the a[%d] value: ",i);
		scanf("%d",&a[i]);
	}
	for(i=0;i<5;i++)
	{
		sum = sum+a[i];
	}
	avg = sum/5;	
	printf("the avg is %f \n",avg);
	
	if(avg >= 65)
		printf("secures grade A \n");
	else if(65>avg && avg >= 50)
		printf("secures grade B \n");
	else if(50>avg && avg >= 35)
		printf("secures grade C \n");
	else
		printf("FAILED \n");
}
